src/Symfony/Bridge/Doctrine/phpunit.xml.dist

```diff
@@ -18,7 +17,7 @@
         </testsuite>
     </testsuites>
 
-    <coverage>
+    <source>
```

Comment:
We need the `ignoreSuppressionOfDeprecations="true"` parameter in the component-level config files as well.

Reply:
done